src/codegen/metadata/attributes.py

```diff
@@ -28949,7 +28949,7 @@
             'is_python_object': False,
             'name': 'START_TRIG_TRIG_WHEN',
             'python_class_name': 'TimeStartTrigger',
-            'python_data_type': 'unknown',
+            'python_data_type': 'datetime',
```

Comment:
I assume you will update the rest of these in a follow-up PR.

Reply:
Correct me if I'm wrong but I think there is not much else to update with StartTrigTrigWhen. If you are referring to the other time trigger attributes in this file then yes, I will update them in a follow up PR.

Reply:
Yes, I'm referring to the other time trigger attributes. Thanks.